lib/generators/deimos/schema_class/templates/schema_record.rb.tt

```diff
@@ -62,7 +62,15 @@
     def namespace
       '<%= @current_schema.namespace %>'
     end
-
+<%- if @tombstone_assignment %>
+    def self.tombstone(key)
+      record = self.new
+<%- if @tombstone_assignment.present? -%>
+      <%= @tombstone_assignment %>
+<%- end -%>
```

Comment:
Is this `if @tombstone_assignment` checking `false`/`true` and the next one checking for `nil`?

Reply:
The first check is for `nil` and the second is for blank string.